portality/models/harvester.py

```diff
@@ -8,7 +8,7 @@ class HarvesterPlugin(object):
     def get_name(self):
         raise NotImplementedError()
 
-    def iterate(self, issn, since, to=None):
+    def iterate(self, issn, since, job, to=None):
```

Comment:
job has been added to the signature, but it doesn't seem to be used anywhere inside the model